tests/test_jinja_macros.py

```diff
@@ -718,3 +719,51 @@ def test_assessment_flag(self, request_ctx):
             r"\d{2}/\d{2}/\d{4} at \d{2}:\d{2}",
             rendered_html,
         ), "Date created not found"
+
+    def test_assessment_completion_state_completed(self, request_ctx):
+        rendered_html = render_template_string(
+            "{{assessment_complete(state, flag, csrf_token, application_id,"
+            " current_user_role)}}",
+            assessment_complete=get_template_attribute(
+                "macros/assessment_completion.html", "assessment_complete"
+            ),
+            state=type("State", (), {"workflow_status": "COMPLETED"})(),
+            flag=None,
+            csrf_token=generate_csrf(),
+            application_id=1,
+            current_user_role="LEAD_ASSESSOR",
+        )
+
+        rendered_html = rendered_html.replace("\n", "")
```

Comment:
We could pass the html into beautiful soup which would make it much easier to parse.

Reply:
Okay 👍

Reply:
@gidsg I have parsed it with beautiful soup. Please review it 🙏